src/odemis/gui/cont/acquisition/cryo_acq.py

```diff
@@ -286,6 +289,29 @@ def _on_export_data_done(self, future):
         data = future.result()
         self._display_acquired_data(data)
 
+    def _add_feature_info_filename(self):
+        """
+        Add feature name, feature status and the counter at the end of the filename.
+        """
+        local_tab = self._tab_data.main.getTabByName("cryosecom-localization")
+        feature_name = local_tab.tab_data_model.main.currentFeature.value.name.value
+        feature_status = local_tab.tab_data_model.main.currentFeature.value.status.value
+        feature_pattern = f"{feature_name}-{feature_status}"
+
+        if 'Feature' in self._config.fn_ptn:
+            # If feature pattern is present in filename
+            # Replace Feature and everything after with the current feature pattern and the updated count
+            base_name = re.sub(r'(Feature).*', f'{feature_pattern}-0', self._config.fn_ptn)
```

Comment:
I'd rather drop this feature. It's too hidden as-is for the user to discover it. For now it's fine to *always* have the same filename modification, irrespective of the filename content.